meta-resin-common/classes/binary-compress.bbclass

```diff
@@ -0,0 +1,28 @@
+# e.g.  COMPRESS_BINARY = "path/bin1 path/bin2 path/bin3"
```

Comment:
The path in the docker recipe is not absolute either, I should rmeove the first / to make it clear.